webrtc-stats.html

```diff
@@ -3016,9 +3016,12 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  The URL of the TURN or STUN server indicated in the that translated this IP
-                  address. It is the URL address surfaced in an
-                  <code>RTCPeerConnectionIceEvent</code>.
+                  For local candidates this is the URL of the ICE server from
+                  which the candidate was obtained. It is the same as the
+                  <a href="https://w3c.github.io/webrtc-pc/#rtcpeerconnectioniceevent">url surfaced in the RTCPeerConnectionIceEvent</a>.
+                </p>
+                <p>
+                  For remote candidates, this property is not applicable.
```

Comment:
Can we s/applicable/present/ ?

Reply:
will change in both places

Reply:
hrm:
```
When a feature is not applicable to an instance of an object (for example audioLevel
          on a video stream), omit the dictionary member. Do NOT report a count of zero, -1 or
          "empty string".
```
We use "applicable" four times (up from 3). 
We do not use it for things like pliCount.

Keep as is an file an issue to be more consistent?